docs/building/unix-instructions.md

```diff
@@ -42,11 +42,12 @@ macOS 10.12 (Sierra) or higher is needed to build dotnet/machinelearning.
 
 On macOS a few components are needed which are not provided by a default developer setup:
 * cmake 3.10.3
-* gcc
+* libomp
+* libgdiplus
+* gettext
```

Comment:
libomp for MKL I assume, libgdiplus for System.Drawing, what gettext for?

Reply:
`libomp` is for `LightGBM`.

`libgdiplus` is for System.Drawing. I guess `gettext` is for `libgdiplus`: https://github.com/mono/libgdiplus/blob/f1dfa1bf87b26dfe74448a3fff9716793a78e98e/README.md#requirements